crowdsourcing/serializers/project.py

```diff
@@ -36,12 +36,15 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    csv_data = serializers.CharField(read_only=True)
```

Comment:
read_only means that this field will be serialized when reading (GET request) i.e it's not used in POST/PUT